### ISSUE TITLE
latest nltk breaks tool, general feedback

### ISSUE THREAD

**Issue by kennell** [drien/tts-joinery#5]
Great tool. Thank you !

1. Latest version of `nltk` (version `3.9.1`) throws a `LookupError` exception, downgrading to version `3.8.1`  fixes it

```
LookupError:
**********************************************************************
  Resource punkt_tab not found.
  Please use the NLTK Downloader to obtain the resource:

  >>> import nltk
  >>> nltk.download('punkt_tab')
```

2. Minor CLI improvement suggestion: apply `.lower()` when parsing the `--voice` param

Cheers

**Comment by kennell** [comment]
one more thing: support for other formats as they seem to be supported by OpenAI now, example: opus, wav, aac

**Comment by drien** [comment]
Hey @kennell, thanks for the report! I've published a new version 1.0.4 that pins nltk to >=3.8.2. It looks like they released a [breaking change](https://github.com/nltk/nltk/issues/3293) to fix a security concern with the original punkt module that I didn't catch because it uses a system-wide cache :disappointed:. I also added auto-lowercasing of the CLI args per your suggestion.

I'll make a new issue for supporting alternate output formats.
